src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -357,8 +366,17 @@ export class SavedObjectsClient {
       }
     });
 
+    const { docs } = response;
+
+    let docsToReturn = docs;
+    if (typeof options.documentFilter === 'function') {
```

Comment:
Do we still need this?

Reply:
I should be able to pull this logic up into the `spaces_saved_object_client` now that we've introduced `extraSourceProperties`

Reply:
awesome, thanks!